Sources/TokamakCore/Gestures/GestureEnvironmentKey.swift

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Szymon on 13/8/2023.
+//
+
+private struct GestureEnvironmentKey: EnvironmentKey {
+    static let defaultValue: GestureEnvironmentValue = GestureEnvironmentValue()
+}
+
+extension EnvironmentValues {
+    var gestureListener: GestureEnvironmentValue {
+        get { self[GestureEnvironmentKey.self] }
+        set { self[GestureEnvironmentKey.self] = newValue }
+    }
+}
+
+class GestureEnvironmentValue {
```

Comment:
```suggestion
final class GestureEnvironmentValue {
```
Would `GestureContext` be a better name for this?
Could you add some doc comments to explain the purpose of this environment value as well?

Reply:
fixed